internal/event/target/nats.go

```diff
@@ -187,6 +187,11 @@ func (n NATSArgs) connectStan() (stan.Conn, error) {
 	}
 
 	connOpts := []stan.Option{stan.NatsURL(addressURL)}
+
+	if n.TLS && !n.TLSSkipVerify {
+		connOpts = append(connOpts, stan.NatsOptions(nats.Secure(&tls.Config{RootCAs: n.RootCAs})))
+	}
```

Comment:
stan is deprecated by NATS folks, please move to JetStream @rayjanoka

Reply:
I would like to but I didn't think minio supported it.

https://github.com/minio/minio/issues/14942

Reply:
It is supported in master @rayjanoka - we deprecated stan since today's release. This PR is not needed.

Reply:
great news! thanks @harshavardhana!